src/ru/skypro/Main.java

```diff
@@ -1,8 +1,56 @@
 package ru.skypro;
 
+import java.util.Arrays;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+	
+        int [] array = new int [3];
+        array [0] = 1;
+        array[1] =2;
+        array[2] = 3;
+
+        float [] arr = {1.57f, 7.654f, 9.986f};
+
+        System.out.println(" ");
+        int [] arrow = {5,10,15,20,25,30};
+
+        System.out.println("Задача 2");
+        System.out.println(array[0] + ", " + array[1] + ", " + array[2]);
+
+        for (int i = 0; i < arr.length - 1; i++) {
+        System.out.print(arr[i] + ", ");}
```

Comment:
проблемы с форматированием кода.
Используйте ctrl+alt+L для автоформатирования
```suggestion
        for (int i = 0; i < arr.length - 1; i++) {
            System.out.print(arr[i] + ", ");
        }
```